README.md

```diff
@@ -1,58 +1,88 @@
 # QuPath Cloud Healthcare API Extension
 
-The **QuPath Cloud Healthcare API** provides viewing whole-slide images online without full downloading and synchronize annotations with the help of [Google Cloud Healthcare API](https://cloud.google.com/healthcare/).
+The **QuPath Cloud Healthcare API** provides viewing whole-slide images online without 
+full downloading and [synchronize annotations](#synchronization-annotations) with the help of 
+[Google Cloud Healthcare API](https://cloud.google.com/healthcare/).
 
 ## Installation:
-1) Download the latest JAR from the releases tab.
-2) Run **QuPath** application and drop the JAR on it's window or just put to /extensions in QuPath data folder.
 
->If you haven't installed QuPath, install QuPath from [here](https://github.com/qupath/qupath/releases/tag/v0.1.2). 
+1) Install QuPath v0.2.0-m2 from [here](https://github.com/qupath/qupath/releases/tag/v0.2.0-m2).
+2) Download the latest JAR from the releases tab.
+3) When you install QuPath, folder for extensions is set */home/user/QuPath/extensions* by default 
+(in Linux). You can put the JAR in this folder or just drag-and-drop it in the QuPath window.
+> For new version another QuPath user directory is recommended to avoid conflict with the previous 
+> extension version. You can set another user directory. Press edit -> preferences and 
```

Comment:
I think it can just be, you can set another user directory by going to Edit -> Preferences -> Set another user directory in Qupath

Reply:
Done